src/pages/docs/guides/chat/build-embedded-chat.mdx

```diff
@@ -0,0 +1,414 @@
+---
+title: "Guide: Building embedded chat experiences with Ably"
+meta_description: "Architecting embedded, contextual chat with Ably for 1:1 and small group conversations linked to a specific context. Performance, reliability, and cost optimization for support tickets, deliveries, gaming sessions, and more."
+meta_keywords: "embedded chat, contextual chat, 1:1 chat, small group chat, realtime messaging, support tickets, delivery tracking, gaming chat, Ably Chat, chat SDK, dependability, cost optimization"
+---
+
+![Embedded chat example mockup](../../../../images/content/chat/ingame-chat.png)
+
+Ably Chat is purpose-built for all types of chat experiences, from 1:1 conversations to small group interactions that enhance specific activities and workflows.
+
+If you need to deliver seamless, contextual chat experiences that enhance tasks like support tickets, deliveries, gaming sessions, collaborative work, or transactional processes, without sacrificing performance, reliability, or costs, then Ably is the proven choice.
+
+Building with Ably means that you no longer need to worry about managing websocket servers, handling failover, network disruptions, or keeping latency low. Ably handles all of this for you, leaving you free to focus on creating engaging experiences.
+
+This guide explains the architectural decisions, technical challenges, and unique benefits of building embedded or contextual chat experiences with Ably, focusing on 1:1 and small group chats. It will help you design contextual, efficient chat solutions for scenarios like customer support, delivery tracking, gaming sessions, collaborative work, marketplace transactions, and more - all with performance, user experience, and cost optimization in mind.
```

Comment:
I don't think you need to re-list all the use cases here as you've done so above.

Reply:
changed the intro to have less repetition